sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -714,7 +714,6 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offersToPr
                 declineOffers(driver, Collections.emptyList(), offers);
                 return;
             }
```

Comment:
Aw, I kind of like blank lines after blocks in most cases, but that's a mega-nit.